include/sys/syscall_lookup.h

```diff
@@ -33,13 +33,13 @@ SYSCALL_LOOKUP(gettid,                     0)
   SYSCALL_LOOKUP(getppid,                  0)
 #endif
 
-SYSCALL_LOOKUP(sched_getparam,             2)
-SYSCALL_LOOKUP(sched_getscheduler,         1)
+SYSCALL_LOOKUP(nxsched_get_param,          2)
+SYSCALL_LOOKUP(nxsched_get_scheduler,      1)
 SYSCALL_LOOKUP(sched_lock,                 0)
```

Comment:
what' rule to add or not add the nx prefix?

Reply:
I think there is no explicit rule. If I understand correctly there was an idea to have a POSIX interface as a syscall layer. But looking into syscall list seems that it is not possible.
I'll convert this PR to draft to keep discussion here.

Reply:
I mean that I'm open to any rules if they are serving common sense and are accepted by others.

Reply:
> what' rule to add or not add the nx prefix?

The idea in the naming was to create a separate (but very similar) similar namespace for functions that should be called only within the OS vs. functions that can be called only from applications.

Reply:
yes I concur make syscalls private ; not forcibly by programming or compiler extension but by tagging like

__nx_private__ ... nxsched_get_param (... )